integration_test/third_party_apps_data/applications/vault/metadata.yaml

```diff
@@ -139,5 +250,38 @@ configuration_options:
     - name: wildcard_refresh_interval
       default: 60s
       description: The interval at which wildcard file paths in include_paths are refreshed. Given as a time duration, for example 30s or 2m. This property might be useful under high logging throughputs where log files are rotated faster than the default interval.
```

Comment:
`include_paths`  (also at line 246, which I couldn't comment on for some reason).